documentation/meta/documentation/quickstart.md

```diff
@@ -5,50 +5,22 @@ locally.
 
 ## Prerequisites
 
-Refer to the [general setup guide](/general/general_setup.md) for setting up the
-prerequisites. Refer to the 'Docs' column in the
-[requirement matrix](/general/general_setup.md#requirement-matrix) to know what
-you need to run this.
+Follow the [general setup guide](/general/general_setup.md) to set up `ov`.
 
 ## Starting up
 
-1. Ensure you download, install and set up all prerequisites.
-
-2. Clone the repository to your computer. Then switch to the cloned directory.
-   If you're planning to contribute, fork the repo and clone your fork instead.
-
-   ```{note}
-   We recommend cloning with the `--filter=blob:none` flag as it dramatically
-   reduces the filesize and download time by creating a "blobless clone".
-   You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
-   ```
-
-   ```bash
-   git clone --filter=blob:none https://github.com/WordPress/openverse.git # or your fork
-   cd openverse/
-   ```
-
-   If you followed the general setup guide and installed
-   [GitHub CLI](/general/general_setup.md#github-cli), you can clone more simply
-   using the `gh` command.
-
-   ```bash
-   gh repo clone WordPress/openverse -- --filter=blob:none  # or your fork
-   cd openverse/
-   ```
-
-3. Install only the Python dependencies. You do not need to install any Node.js
+1. Install only the Python dependencies. You do not need to install any Node.js
    dependencies to run the documentation.
 
    ```bash
-   ./ov just documentation/install
+   ov just documentation/install
    ```
 
-4. Run the documentation live server. Once this is done, you should be able to
+1. Run the documentation live server. Once this is done, you should be able to
    see the documentation on [http://127.0.0.1:50230](http://127.0.0.1:50230).
 
    ```bash
-   ./ov just documentation/live
+   ov just documentation/live
```

Comment:
I tried this, but I cannot see the docs locally, not even after fixing the few lost references. Do you know what might be happening? The docs are simply not found.

Reply:
Have you run `pdm install` inside the documentation site? I'll update the instructions in case they don't include that. `cd documentation && ov pdm install` or `ov pdm install -p documentation` should work.

Reply:
Shouldn't `ov just documentations/install` be enough? Those commands didn't make any change tho.

Reply:
It should be. What's the specific output you're getting when running `ov just documentation/live`? Are you running it from the repository root or from inside the documentation directory?

Reply:
I'm also getting an error here:
```
$ ov just documentation/install
pdm install 
WARNING: Lockfile is generated on an older version of PDM
WARNING: Project requires a python version of ==3.11.*, The virtualenv is being created for you as it cannot be matched to the right version.
INFO: python.use_venv is on, creating a virtualenv for this project...
See /opt/.local/state/pdm/log/pdm-install-resolve-q3x9ldrz.log for detailed debug log.
[PermissionError]: [Errno 13] Permission denied: '/opt/pdm/venvs'
WARNING: Add '-v' to see the detailed traceback
error: Recipe `install` failed on line 16 with exit code 1
```

Reply:
@AetherUnbound your problem can be fixed by running `ov sudo chmod -R 0777 /opt`.

That cannot be the issue @krysal is having because permissions problems only happen to us Linux users (macOS users are running `ov` as root at all times, so there aren't any file permissions considerations for them at all).

I'll add a recipe `ov doctor` that runs `sudo chmod -R 0777 /opt` and can be expanded to run other fixes when there are weird issues like this.

@krysal can you let me know what the problem you ran into is, specifically? I'd really like to get this PR merged...

Reply:
@AetherUnbound if you re-pull the branch you can try out `ov doctor`.